src/Control/Eff/Gitter.hs

```diff
@@ -0,0 +1,37 @@
+{-
+    Notifaika reposts notifications
+    from different feeds to Gitter chats.
+    Copyright (C) 2015 Yuriy Syrovetskiy
```

Comment:
The Author is the only copyright holder (it's my religion).

I can't find any clarification on which author must be placed in this line, but the only logical for me it seems to be the author of this particular file.

Also, 2016.